server/route.go

```diff
@@ -967,7 +965,7 @@ func (s *Server) updateRemoteRoutePerms(c *client, info *Info) {
 
 	c.mu.Lock()
 	c.sendRouteSubProtos(allSubs, false, func(sub *subscription) bool {
-		subj := string(sub.subject)
+		subj := bytesToString(sub.subject)
```

Comment:
Again, canExport()/canImport() is same than canSubscribe() and will do sl.Match(), so we can't do that.